context/app/static/js/shared-styles/panels/IconPanel/style.ts

```diff
@@ -0,0 +1,8 @@
+import { SvgIconProps } from '@mui/material/SvgIcon';
+import { styled } from '@mui/material/styles';
+
+export const getStyledIcon = (IconComponent: React.ComponentType<SvgIconProps>) =>
+  styled(IconComponent)(({ theme }) => ({
+    color: theme.palette.primary.main,
+    fontSize: '1.5rem',
+  }));
```

Comment:
We should be able to get rid of this and apply styles to `SeverityIcon` directly.